Tk example images aren't in Tk's default example install dir.


Index: tclpkg/tcldot/demo/doted.tcl
--- tclpkg/tcldot/demo/doted.tcl.orig
+++ tclpkg/tcldot/demo/doted.tcl
@@ -20,7 +20,7 @@ proc mouse_anyenter {c} {
 	set item [string range [lindex [$c gettags current] 0] 1 end]
 	set saveFill [list $item [lindex [$c itemconfigure 1$item -fill] 4]]
 	$c itemconfigure 1$item -fill black \
-		-stipple @$tk_library/demos/images/gray25.xbm
+		-stipple @${LOCALBASE}/share/examples/tk${MODTK_VERSION}/images/gray25.xbm
 }
 
 # as the mouse moves out of an object restore its shading
@@ -613,7 +613,7 @@ button .b.layout \
 	-height [.b.h cget -width] \
 	-foreground green \
 	-activeforeground green\
-	-bitmap @$tk_library/demos/images/gray25.xbm \
+	-bitmap @${LOCALBASE}/share/examples/tk${MODTK_VERSION}/images/gray25.xbm \
 	-command "layout $c"
 
 # initialize zoom state
